### ISSUE TITLE
Support 'icon_prop_key' entry in ibus-mozc

### ISSUE THREAD

**Issue by yukawa** [google/mozc#344]
Environment
-------------------------
ibus-mozc 2.17.2287.103 running on Fedora 23 KDE Plasma Desktop.

Steps to reproduce:
-------------------------
 1. Build and install ibus-mozc 2.17.2287.103.
 2. Make sure that ibus-mozc is selected as the current input method.
 3. Make sure that the setting ```Show icon on system tray``` is enabled on the ```ibus-setting``` dialog.
 4. Focus in any text field.

Expected behavior:
-------------------------
Status icon such as ```あ``` and ```A``` is displayed on the KDE5 system tray.

### Hiragana Mode
![hiragana](https://cloud.githubusercontent.com/assets/5861397/12078188/ba63bf86-b1bc-11e5-9271-9c9c89614785.png)

### Katakana Mode
![katakana](https://cloud.githubusercontent.com/assets/5861397/12078189/bc718970-b1bc-11e5-93ba-603c1a0512e0.png)

### Direct Mode
![off](https://cloud.githubusercontent.com/assets/5861397/12078190/bddc2932-b1bc-11e5-9879-eda982f5d231.png)

Actual behavior:
-------------------------
Mozc's product icon ![ime_product_icon_opensource-32.png](https://raw.githubusercontent.com/google/mozc/ec8a7bf5d761c0704be4920f11332c9ce9696184/src/data/images/unix/ime_product_icon_opensource-32.png) is displayed on the KDE5 system tray.

![actual](https://cloud.githubusercontent.com/assets/5861397/12078197/ff8e4108-b1bc-11e5-96e3-f1acdd1dddd1.png)

Additional Information:
-------------------------
Showing a status icon instead of the IME's product icon is a new feature introduced by ibus/ibus@23c45b970b195008a54884a1a9d810e7f8b22c5c and has been available since IBus 1.5.11.

Basically all we need to do is just to add ```<icon_prop_key>``` entry in ```mozc.xml``` when IBus is 1.5.11 or greater.  Although supporting new IBus feature is not our goal any more, doing this before we delete ibus-mozc in #287 may help some people.
